apps/astarte_realm_management/lib/astarte_realm_management/queries.ex

```diff
@@ -19,6 +19,11 @@
 defmodule Astarte.RealmManagement.Queries do
   require CQEx
   require Logger
+  alias AstarteRealmManagement.Realms.DeletionInProgress
+  alias Astarte.RealmManagement.Realms.GroupedDevice
+  alias Astarte.RealmManagement.Realms.Name
```

Comment:
I am missing the point of missing of a point.

```suggestion
  alias Astarte.RealmManagement.Realms.DeletionInProgress
  alias Astarte.RealmManagement.Realms.GroupedDevice
  alias Astarte.RealmManagement.Realms.Name
```